docs/web/site/docs.css

```diff
@@ -60,6 +60,51 @@
   src: url('tenor-sans-v12-latin-regular.woff2') format('woff2');
 }
 
+/* Theme */
+
+/* Dark theme (default) */
+:root {
+  --bg-color: #131618;
+  --text-color: #c9d1d9;
+  --pre-bg-color: #1a1f26;
+  --code-bg-color: #2c333b;
+  --border-color: #282828;
+  --h1-text-shadow: black;
```

Comment:
This feels questionable.

Reply:
Removed incorrect references to `h1-text-shadow` in [741a597](https://github.com/aantron/dream/pull/268/commits/741a59739680bca79e0f9939422ead7803f5f3d8)